rpc/params/apierror.go

```diff
@@ -373,6 +373,10 @@ func IsCodeNotFound(err error) bool {
 	return ErrCode(err) == CodeNotFound
 }
 
+func IsCodeNotValid(err error) bool {
+	return ErrCode(err) == CodeNotValid
+}
```

Comment:
What's this for?

Reply:
I use it in some tests, for some reason it was left out